src/component/app.jsx

```diff
@@ -1,43 +1,23 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
-import {
-    Layout,
-    Drawer,
-    Header,
-    Navigation,
-    Content,
-    Footer,
-    FooterSection,
-    FooterDropDownSection,
-    FooterLinkList,
-    Grid,
-    Cell,
-    Icon,
-} from 'react-mdl';
-import { Link } from 'react-router';
+import { Layout, Header, Navigation, Content, Footer, Grid, Cell } from 'react-mdl';
+import { Route, Redirect, Switch } from 'react-router-dom';
 import styles from './styles.scss';
 import ErrorContainer from './error/error-container';
 
 import AuthenticationContainer from './user/authentication-container';
 import ShowUserContainer from './user/show-user-container';
 import ShowApiDetailsContainer from './api/show-api-details-container';
-import { ScrollContainer } from 'react-router-scroll';
-
-function replace(input, params) {
-    if (!params) {
-        return input;
-    }
-    Object.keys(params).forEach(key => {
-        input = input.replace(`:${key}`, params[key]);
-    });
-    return input;
-}
+import Features from '../page/features';
+import { DrawerMenu } from './menu/drawer';
+import { FooterMenu } from './menu/footer';
+import Breadcrum from './menu/breadcrumb';
+import { routes } from './menu/routes';
 
 export default class App extends Component {
     static propTypes = {
```

Comment:
You should be able to install https://github.com/flow-typed/flow-typed/blob/master/definitions/npm/react-router-dom_v4.x.x/flow_v0.63.x-/react-router-dom_v4.x.x.js (`flow-typed install react-router-dom@4.x.x`) to get more accurate typing

Reply:
Ahh thanks, I will check it out!

Reply:
Because I am using Visual Studio Code I already got the ts definitions for [react-router-dom](https://github.com/DefinitelyTyped/DefinitelyTyped) automatically in the IDE, can't see that these flow types will add much more?